torchtune/datasets/_wikitext.py

```diff
@@ -47,12 +47,29 @@ def wikitext_dataset(
         split (str): ``split`` argument for ``datasets.load_dataset``. You can use this argument to load a subset
             of a given split, e.g. ``split="train[:10%]"``. Default is "train".
         **load_dataset_kwargs (Dict[str, Any]): additional keyword arguments to pass to ``load_dataset``.
-
+    Raises:
+        ValueError: If ``packed=True`` and ``tokenizer.max_seq_len`` is not set.
     Returns:
         Union[TextCompletionDataset, PackedDataset]: the configured :class:`~torchtune.datasets.TextCompletionDataset`
             or :class:`~torchtune.datasets.PackedDataset` if ``packed=True``
     """
 
+    if packed:
+        if tokenizer.max_seq_len is None:
```

Comment:
This error is already checked and raised in `text_completion_dataset` so you don't need to check it again here. Should be sufficient to just pass `packed` directly through

Reply:
Oh I see. Will be fixed soon.

Reply:
Probably now is fine

Reply:
Should be fine now